charts/sscs-tribunals-api/values.preview.template.yaml

```diff
@@ -1,8 +1,10 @@
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  aadIdentityName:
   environment:
     PDF_API_URL: "http://cmc-pdf-service-aat.service.core-compute-aat.internal"
     CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
-    IDAM_OAUTH2_REDIRECT_URL: "https://evidence-sharing-preprod.sscs.reform.hmcts.net"
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
+    IDAM_S2S_AUTH: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
```

Comment:
We don't need these i believe? 
```suggestion
```

Reply:
we do as env would be set to preview in the preview env?

Reply:
It is set to aat in jenkins as we read aat vaults for preview.